justfile

```diff
@@ -71,6 +71,12 @@ launch-py:
     cargo run --quiet -- show
     ls *.py | entr -p cargo run --quiet -- run --command "python3 tmp.py"
 
+launch-c:
+    code tmp.c
```

Comment:
Maybe we should use $EDITOR or some other paradigm? Hardcoding VS Code seems short sighted

Reply:
That seems simple to add. Will do.